test/PidTest.cpp

```diff
@@ -0,0 +1,192 @@
+/**
+ * @file PidTest.cpp
+ * @brief PidGainクラス,Pidクラスをテストする
+ * @author Negihara
+ */
+
+#include "Pid.h"
+#include <gtest/gtest.h>
+
+namespace etrobocon2023_test {
+  TEST(PidGainTest, gain)
+  {
+    double expected_p = 0.1;
```

Comment:
これも去年統一できてなかったんだけど、テストコードでも、変数名はローワーキャメルケース(小文字始まりの大文字つなぎ)にしたい。

```suggestion
    double expectedP = 0.1;
```